db/compaction_picker_universal.cc

```diff
@@ -568,6 +568,12 @@ Compaction* UniversalCompactionPicker::PickCompactionToReduceSortedRuns(
     output_level = sorted_runs[first_index_after].level - 1;
   }
 
+  // last level is reserved for the files ingested behind
+  if (ioptions_.allow_ingest_behind &&
+      (output_level == vstorage->num_levels() - 1)) {
+    output_level--;
```

Comment:
i think we only that logic to be triggered if there'are at least 3 levels, then we put it to l1 instead of l2; but never on l0, we don't want them on l0. in other words, we want that if we allow ingest behind there're num_levels >=3 ?